concent_api/core/tests/test_unit_model.py

```diff
@@ -214,24 +212,6 @@ def setUp(self):
         self.client.clean()
         self.client.save()
 
-    def test_that_exception_is_raised_when_ethereum_address_has_wrong_length(self):
```

Comment:
Hm... why do we want to remove these?

Reply:
Because this is no longer our custom validation, but we are using framework functionality, and we don't want to test framework functionality, as it has its own tests.

Reply:
that was a tricky question :)